<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreviewFrame</class>
 <widget class="QWidget" name="PreviewFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>711</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="tabletTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QTreeView" name="treeView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>230</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragOnly</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::LinkAction</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="VideoPanel" name="videoPanel" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbBottomTool">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="tbOpenAll">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="toolTip">
             <string>打开所有</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/qrc.qrc">
              <normaloff>:/images/openvideos.png</normaloff>:/images/openvideos.png</iconset>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tbCloseAll">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="toolTip">
             <string>关闭所有</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/qrc.qrc">
              <normaloff>:/images/closevideos.png</normaloff>:/images/closevideos.png</iconset>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tbFullScreen">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="toolTip">
             <string>全屏</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/qrc.qrc">
              <normaloff>:/images/fullscreen.png</normaloff>:/images/fullscreen.png</iconset>
            </property>
            <property name="popupMode">
             <enum>QToolButton::DelayedPopup</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
            <property name="arrowType">
             <enum>Qt::NoArrow</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tbCutWindow">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="toolTip">
             <string>显示窗体分割</string>
            </property>
            <property name="statusTip">
             <string/>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/qrc.qrc">
              <normaloff>:/images/windowcut.png</normaloff>:/images/windowcut.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="popupMode">
             <enum>QToolButton::DelayedPopup</enum>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonIconOnly</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
            <property name="arrowType">
             <enum>Qt::NoArrow</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/qrc.qrc">
              <normaloff>:/images/setting1.png</normaloff>:/images/setting1.png</iconset>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="../resources/qrc.qrc">
     <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
   </property>
   <property name="text">
    <string>添加节点</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../resources/qrc.qrc">
     <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
   </property>
   <property name="text">
    <string>删除节点</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="../resources/qrc.qrc">
     <normaloff>:/images/play.png</normaloff>:/images/play.png</iconset>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../resources/qrc.qrc">
     <normaloff>:/images/stop.png</normaloff>:/images/stop.png</iconset>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </action>
  <action name="action_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/qrc.qrc">
     <normaloff>:/images/windowcut_1.png</normaloff>:/images/windowcut_1.png</iconset>
   </property>
   <property name="text">
    <string>界面1</string>
   </property>
  </action>
  <action name="action_4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/qrc.qrc">
     <normaloff>:/images/windowcut_4.png</normaloff>:/images/windowcut_4.png</iconset>
   </property>
   <property name="text">
    <string>界面4</string>
   </property>
  </action>
  <action name="action_6">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/qrc.qrc">
     <normaloff>:/images/windowcut_6.png</normaloff>:/images/windowcut_6.png</iconset>
   </property>
   <property name="text">
    <string>界面6</string>
   </property>
  </action>
  <action name="action_9">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/qrc.qrc">
     <normaloff>:/images/windowcut_9.png</normaloff>:/images/windowcut_9.png</iconset>
   </property>
   <property name="text">
    <string>界面9</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoPanel</class>
   <extends>QWidget</extends>
   <header location="global">videopanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/qrc.qrc"/>
 </resources>
 <connections/>
</ui>
